browser/components/MarkdownPreview.js

```diff
@@ -133,22 +133,67 @@ body p {
 `
 }
 
+//
+// apply style to the scroll bar for 
+// pre view and split view
+// 
 const scrollBarStyle = `
+::-webkit-scrollbar-track {
+  background-color: #FFFFFF;
+}
+
+::-webkit-scrollbar {
+  width: 12px;
+  background-color: #FFFFFF;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: #DDDDDD; // $ui-dark-text-color = #DDDDDD ?
```

Comment:
Please remove the comments or change to /**/ as these code will be injected to iframe as CSS and CSS doesn't have // comment :smile: